### ISSUE TITLE
Add an list of worlds a module is enabled in to the Admin Screen

### ISSUE THREAD

**Issue by MrVauxs** [foundryvtt/foundryvtt#10628]
### User Experience

It would be great to be able to see what modules are enabled where in the admin screen.

Here is a mockup
![chrome_sn9GJnT9Ir](https://github.com/foundryvtt/foundryvtt/assets/32039708/3ceff140-3bbc-427a-babd-91992509f79c)

**Comment by aaclayton** [comment]
Good idea, but part of the reason we haven't done this is that the active module configuration for a world is stored within that world's database, so we would need to connect to the DB for each world in order to load this list. That's not impossible, but it's _a reason_ why we did not do this in the past.

**Comment by MrVauxs** [comment]
If that is the case then I think there is a lot more that could be done with caching world data. Seeing proper statistics about their game worlds, like for example: how many modules a user has enabled, would certainly give people a chance to better consider how they take care of Foundry!

**Comment by aaclayton** [comment]
Comment from @ian612

On the setup page on the Add-on Modules tab, it would be nice if there was a way to see the date a module was originally added, the date it was last updated (as in when the latest version was downloaded, not authored), and whether a module is active in any worlds, either via mouseover tooltip or via the context menu.

Every so often I go to that tab to clean up any old modules that I don't use anymore, but I don't always remember when I installed them or which I still need (especially for games that are on hiatus or that are played infrequently). This would help prevent accidental deletion of modules I think I don't need, but actually still do.
